### ISSUE TITLE
Programmatically add alt text for book covers

### ISSUE THREAD

**Issue by SteelWagstaff** [pressbooks/pressbooks-book-directory-fe#442]
All book cover images should have alt text which reads `BOOK TITLE cover`

**Comment by SteelWagstaff** [comment]
Works as expected on dev & staging
